src/commands/aksOpenKubectlPanel/aksOpenKubectlPanel.ts

```diff
@@ -45,8 +45,8 @@ export async function aksOpenKubectlPanel(_context: IActionContext, target: unkn
         return;
     }
 
-    if (cluster.result === false) {
-        vscode.window.showErrorMessage("No cluster selected. Please select a valid cluster.");
+    if (cluster.result === true) {
```

Comment:
this is like doing total opposite of what was there. is this intentional? @kejatura-dev

Reply:
So the return on the selected cluster for SelectClusterOptions.NewCluster returns a true value (since return type of cluster can be of ClusterPreference | boolean). If this value is false, this would cause undefined cluster option in the flow.

Reply:
Left a comment in code to explain